src/main/java/com/smalaca/messagesender/service/UserCrud.java

```diff
@@ -20,30 +27,61 @@ public UserCrud(InMemoryUserRepository userRepository) {
         this.userRepository = userRepository;
     }
 
-    public boolean createUser(UserDto userDto) {
+    public Response createUser(UserDto userDto) {
         User user = new UserFactory().createFrom(userDto);
 
-        if ((user.getLogin().equals("")) || user.getEmail().equals("") && user.getTwitter().equals("") && user.getSlack().equals(""))
-            return false;
-
         try {
+            isUserValid(user);
             userRepository.add(user);
-            return true;
+            return aSuccessfulResponseWith("User Created");
         } catch (UserAlreadyExistException e) {
-            return false;
+            return aFailureResponse("User Creation Failed!");
+        } catch (InvalidUserPassed e) {
```

Comment:
do walidacji wystarczy Ci response. To nie jest błąd, że dane są niepoprawne. To jest normalny scenariusz:
```java
if (isValid(user)) {
   // do what you have to do
} else {
  return aFailureResponse("Invalid data given.");
}